index.html

```diff
@@ -12,19 +14,32 @@
       crossorigin="anonymous"
       referrerpolicy="no-referrer"
     />
+    
     <title>Portfolio</title>
 </head>
 <body>
     <header>
+       <div id="menu-">
+      <div id="closed">
+        <span class="close">&times;</span>
+      </div>
+      <div id="menu-list">
+        <ul id="mlist">
+          <li><a href="#work-section">Portfolio</a></li><hr>
+           <li><a href="#about-sec">About</a></li><hr>
+          <li><a href="#contact-form">Contact</a></li> <hr>
+        </ul>
+      </div>
+    </div>
         <nav class="navbar">
           <div class="logo" href="#">Maryam</div>
           <ul class="navbar-flex">
             <li class="nav-link"><a href="#">Portfolio</a></li>
             <li class="nav-link"><a href="#">About</a></li>
             <li class="nav-link"><a href="#">Contact</a></li>
             <li class="envelope"><a href="#"></a></li>
-            <li class="mob-bars">
-              <a href="#"><i class="fa-solid fa-bars"></i></a>
+            <li class="mob-bars" id="mob-barss">
+              <a href="#" id="menu-icon"><i class="fa-solid fa-bars"></i></a>
             </li>
```

Comment:
- [ ] As required for the project, kindly ensure that your design sticks as much as possible to the one from your chose **[Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48-27&t=sbMjA26HZqyHFsYZ-0)** template.
![Screenshot from 2023-04-10 12-34-18](https://user-images.githubusercontent.com/83042742/230947700-70c67cbb-9ab4-4f52-8379-33e01a308208.png)